internal_ws/ykbh/src/lib.rs

```diff
@@ -326,11 +374,42 @@ impl SIRInterpreter {
             Terminator::Goto(bb) => {
                 self.frames.last_mut().unwrap().bbidx = *bb;
             }
+            Terminator::Assert {
+                cond,
+                expected,
+                target_bb,
+            } => {
+                let b = if self.read_int(cond) == 1 {
+                    true
+                } else {
+                    false
+                };
+                if b != *expected {
+                    todo!("Raise error")
```

Comment:
Laurie would ask us to drop the message.

Reply:
I initially has a comment "FIXME raise error", but that's 2 lines vs 1.

Reply:
I think there's a difference where it's clear what the "todo" is, whereas here I think it's not immediately obvious.

Reply:
You could do: `todo!(); // FIXME ...`?

Reply:
I don't really follow the reasoning behind this to be honest.

Reply:
It just reduces the binary size, as one less static string needs to be interned.

FWIW, I prefer the immediacy of the message.

Reply:
It also makes it easier to find the `todo!` within a macro, as the line number in the error message isn't helpful.

Reply:
Funnily enough, I didn't update the `todo!` in the macro. I'm gonna update both: remove the string from this one, but add a string to the one in the macro.

Reply:
See 77d34dc.

Reply:
In general I think that strings in `todo` are pointless, but there are always exceptions. In this case, I think the *current* string is pointless but @ptersilie makes a good point that debugging macros in Rust is hellish. *If* the string was something like `todo!("{:?}"` then it would make sense, because now it genuinely improves readability.